src/components/common/ProductBox/ProductBox.js

```diff
@@ -44,18 +53,46 @@ const ProductBox = ({ name, price, promo, stars, photoBackground }) => (
     <div className={styles.line}></div>
     <div className={styles.actions}>
       <div className={styles.outlines}>
-        <Button variant='outline'>
-          <FontAwesomeIcon icon={faHeart}>Favorite</FontAwesomeIcon>
-        </Button>
-        <Button variant='outline'>
-          <FontAwesomeIcon icon={faExchangeAlt}>Add to compare</FontAwesomeIcon>
-        </Button>
+        {like && (
+          <Button variant='outline'>
+            <FontAwesomeIcon icon={faHeart} className={styles.like}>
+              Favorite
+            </FontAwesomeIcon>
+          </Button>
+        )}
+
+        {!like && (
+          <Button variant='outline'>
+            <FontAwesomeIcon icon={faHeart}>Favorite</FontAwesomeIcon>
+          </Button>
+        )}
+
+        {compare && (
+          <Button variant='outline'>
+            <FontAwesomeIcon icon={faExchangeAlt} className={styles.compare}>
+              Add to compare
+            </FontAwesomeIcon>
+          </Button>
+        )}
+
+        {!compare && (
```

Comment:
Ta linia i 70 różnią się tylko dodaniem klasy lub nie, nie ma potrzeby tworzyć HTML tego samego 2 razy, zrób ifa tylko przy `className` to samo tyczy się like